modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -2,17 +2,23 @@
 
 Use the following procedures to back up and restore Runtime Fabric services and the required state necessary for all deployed applications and their configurations.
 
+Data about the state of Runtime Fabric is distributed across the Kubernetes cluster. To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup on external storage outside the Runtime Fabric cluster.
+
+For Runtime Fabric on VMs / Bare Metal, the back up and restore process includes backing up and restoring appliance-related custom resources such as logforwarder and smtp. The back up and restore process does not include backing up metrics data stored in InfluxDB that is generated by the applications.
+
+== When to Use the Back Up and Restore Process
+
 Common scenarios for using the back up and restore process include: 
 
 * Performing a major infrastructure upgrade, such as migrating nodes in a Runtime Fabric cluster to a new version of an operating system.
 * Upgrading Runtime Fabric for VMs / Bare Metal. In some cases, you may want to back up your cluster, install a new cluster using the new Runtime Fabric version, and then restore your backup on the new cluster.
```

Comment:
@kmendiratta-crm does this contradict the new warning? Should we clarify here that you can't run them concurrently? Or is there a difference between BYOK and appliance?

Reply:
should I move the note here?

Reply:
@sarjeet2013

Reply:
In my opinion its not confusing but may be in the warning we can explicitly include this:
In the warning section:
`not back up Runtime Fabric A, use that back up to create a new Runtime Fabric B while the Runtime Fabric A is till active,`.

OR

`Upgrading Runtime Fabric for VMs / Bare Metal. In some cases, you may want to back up your cluster, install a new cluster using the new Runtime Fabric version ON THE SAME CLUSTER, and then restore your backup on the new cluster.`

Reply:
@hegderohit89 I actually took both of these suggestions. Do you want to approve?